OpenSim/Analyses/StaticOptimizationTarget.cpp

```diff
@@ -479,7 +479,8 @@ objectiveFunc(const Vector &parameters, const bool new_parameters, Real &perform
     int na = _model->getActuators().getSize();
     double p = 0.0;
     for(int i=0;i<na;i++) {
-        p +=  pow(fabs(parameters[i]),_activationExponent);
+        if (ScalarActuator* act = dynamic_cast<ScalarActuator*>(&_model->getActuators().get(i)))
```

Comment:
> What would you propose? Unwind that capability?

I don't know. Promote as much functionality up to Actuator to avoid the need for downcasting. It's a separate issue; I don't want to distract too much from the content of this PR.